### DISCUSSION TITLE
What if

### DISCUSSION THREAD

**Discussion by Ybouali** [community/community#86638] [Other Feature Feedback, Questions, & Ideas]
We create a AI that will accept as a input a subject or a word like dinner that is the word the ai will keep asking us questions about the dinner until the AI get as what we want 

I woud like to hear what do u think about this

**Comment by mishmanners** [comment]
Currently ChatGPT can do this. How would your system differ in terms of what it can provide?

**Reply by Ybouali** [reply]
Okay this new ai will provide u with different suggestions.  I have been giving a example of dinner so i will explain more with the same example 
The ai will provide me with soups or whatever is popular in my country or is popular in that season of year 

I don't know if ChatGpt can do that??

**Reply by mishmanners** [reply]
GitHub Copilot can already do this:
https://dev.to/github/how-to-get-more-suggestions-from-github-copilot-12i2
and the video here:
https://youtu.be/VwxYD7YzhvQ

**Reply by Ybouali** [reply]
help me found something new in this world because every time i came up with something u give me something that already exist

help me here

HHHHHHHHHHHHHHHHHHHHHHHH

anyway wait for me i'll find something new

**Reply by Ybouali** [reply]
hmmmmmmm why this ai dose not do both of chatGPT and copilot i know it will be a lot of work but it just an idea

**Comment by Andrkenya** [comment]
> We create a AI that will accept as a input a subject or a word like dinner that is the word the ai will keep asking us questions about the dinner until the AI get as what we want
> 
> I woud like to hear what do u think about this

Based on my view, I believe this type of recommender AI can be implemented. Though it might require integration with a some external parties.

**Reply by Ybouali** [reply]
Explain more please

**Reply by RodmanJrBubs** [reply]
Can I help?

**Reply by Ybouali** [reply]
Go and don't ask

**Comment by yazan-metax** [comment]
This AI concept, which focuses on generating questions about a specific subject such as 'dinner', is intriguing and shows great potential for various applications. By continuously asking questions based on the user's input, the AI can delve deeper into the topic, potentially uncovering nuances and details that might not be immediately apparent.
so I think it is a cool idea, but for sure more info needed

**Reply by Ybouali** [reply]
I am glad that someone understands me and this one from my network so i am really happy,  i don't wanna say that others don't 
understand  no but this one is from my network the 42 network,  yess brother i really need more information about this ai

**Comment by Ybouali** [comment]
I don't know but i think if this ai just as first version will use the chatgpt api it will be helpful just for testing purposes 😉 
And of course get the data from it because we all know it too much money to invest in this crazy idea and the more important will takes time

**Comment by samus-aran** [comment]
Hi @Ybouali , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! 

We want to make sure you’re getting the best support you can now that GitHub Universe has ended so I've gone ahead and moved this to the most relevant category for you.

Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by Ybouali** [reply]
thanks but i did nothing

**Comment by ImanAdithya** [comment]
if is condition
